docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+- [Creating SR-IOV Compute Nodes in IPI](#creating-sr-iov-compute-nodes-in-ipi)
+- [Creating SR-IOV Compute Nodes in UPI](#creating-sr-iov-compute-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using single root I/O virtualization (SR-IOV) networking as an additional network in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
+in OpenShift, and that your tenant has access to them.
+
+Deploying compute nodes with SR-IOV networks is supported as a day 2, or post install, operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, and have succesfully installed an OpenShift cluster that meets the [minimum requirements](README.md#openstack-requirements), you can follow these steps and example workflows to create compute nodes with SR-IOV NICs.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator.
+
+Next, you will need to create a machine config that enables the SR-IOV operator to get the UUID of networks from OpenStack. This machine config will allow the OpenStack metadata service to be available as a mountable drive.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 20-mount-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - name: create-mountpoint-var-config.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create mountpoint /var/config
+            Before=kubelet.service
+
+            [Service]
+            ExecStart=/bin/mkdir -p /var/config
+
+            [Install]
+            WantedBy=var-config.mount
+
+        - name: var-config.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/config
+            What=/dev/disk/by-label/config-2
+            [Install]
+            WantedBy=local-fs.target
+```
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
```

Comment:
drop the "you can apply" and just use apply, unless enabling the no-IOMMU is optional.

Reply:
I copy pasted this directly from the downstream actually. @atyronesmith do you know if this is optional?